evaluate_library_access_from_output_tsv/evaluate_library_access_from_output_tsv.Rmd

```diff
@@ -5,24 +5,30 @@ date: "2017"
 output: pdf_document
 ---
 
-```{r setup, include=FALSE}
-knitr::opts_chunk$set(echo = TRUE)
-knitr::opts_chunk$set(include = FALSE)
-knitr::opts_chunk$set(results = "asis")
-knitr::opts_chunk$set(cache = TRUE)
-```
-
-```{r settings}
-lzma_compressed_library_access_tsv_location <- "data/library_coverage_xml_and_fulltext_indicators.tsv.xz"
+```{r settings, include = FALSE}
+lzma_compressed_library_access_tsv_location <- file.path(
+  'data', 'library_coverage_xml_and_fulltext_indicators.tsv.xz'
+)
 
 original_dataset_with_oa_color_column_location <- paste0(
```

Comment:
This can go?

Reply:
Ah, yes, I forgot to change that.

I can / forgot just now to add on a second table to the Rmd that doesn't stratify by `oadoi_color`, but I actually do want to keep the existing sub-stratified table here; it's something that my supervisor specifically requested to help diagnose whether there are issues we need to debug in our catalog website.

So, I do need that for my own work; but I *can* take it out of this repo., if you prefer. Would you accept me adding a second table that doesn't stratify, and keeping this one in?

Reply:
> Would you accept me adding a second table that doesn't stratify, and keeping this one in?

Sure this Rmd file can be used for these exploratory analyses.

Reply:
The `Rmd` file will also need to be updated to reflect the on- vs. off-campus columns I just added, too, actually. So maybe we could hammer out *exactly* what you'd like for incorporating into the manuscript now.

[When we last talked about it](https://github.com/greenelab/library-access/pull/8#issuecomment-340540079) (I *think* that was the last time we discussed it), you mentioned that you'd like to incorporate the rates we get into "as bars in [Figure 8B](https://greenelab.github.io/scihub-manuscript/#fig:state-of-oa)." Is that still the case? If so, would you want a table along these lines, or something different?

| Sample         | On-Campus                | Off-Campus |
|----------------|--------------------------|------------|
| Web of Science | X out of Y articles (Z%) |            |
| Unpaywall      |                          |            |
| Crossref       |                          |            |

Reply:
Or, I could remove the Rmd file from this PR for now, and we could work on that later.

Reply:
Sorry, also, GitHub didn't auto-refresh, so I didn't see [your comment ](https://github.com/greenelab/library-access/pull/17#discussion_r157878349) before posting follow-ups. Thanks!

Reply:
> Or, I could remove the Rmd file from this PR for now, and we could work on that later.
￼
That makes sense!

> would you want a table along these lines

Let me think a little more about how to respresent the accuracy analysis results. They'll probably go in the methods section. Will tag you in the relevant issue in another repo when the time comes.

Reply:
Sure, that all sounds good to me!